packages/apps/shopify-app-remix/src/server/types.ts

```diff
@@ -257,10 +257,7 @@ interface Authenticate<Config extends AppConfigArg> {
    * };
    * ```
    */
-  webhook: AuthenticateWebhook<
-    RestResourcesType<Config>,
-    keyof Config['webhooks'] | MandatoryTopics
-  >;
+  webhook: AuthenticateWebhook<RestResourcesType<Config>, string>;
```

Comment:
Does this leave behind an unused import to `MandatoryTopics`?

Reply:
No, it's still used here: 
https://github.com/Shopify/shopify-app-js/blob/0154c33e2a0d0ba681eed74fada0f0df2b34cf57/packages/apps/shopify-app-remix/src/server/authenticate/webhooks/authenticate.ts#L18

Reply:
Right, I think we can do it now: https://github.com/Shopify/shopify-app-js/pull/1051/commits/0b5a666acb020c94aefdfa1a575bab307b38ab63